### ISSUE TITLE
Adjust cid enriched geojson so datetime and cid go inside properties

### ISSUE THREAD

**Issue by jsolly** [easierdata/web3-geo-dashboard#26]
## Context 
Currently the cid enriched geojson looks like this:
```json
	"features": [
		{
			"type": "Feature",
			"properties": {
				"PATH": 13,
				"ROW": 27
			},
			"geometry": {
				"type": "Polygon",
				"coordinates": [
					[
						[-72.34003371649325, 46.874317393516314],
					]
				]
			},
			"cid": "QmZ3hux2mwyVZz1nhsLTF8EorPHsHEzWExhdP4pge1q9mr",
			"datetime": "2022-10-17T15:35:43.937866Z"
		},
	]
```

## More correct version
```json
        "features": [
	        {
		        "type": "Feature",
		        "properties": {
			        "PATH": 13,
			        "ROW": 27,
			        "cid": "QmZ3hux2mwyVZz1nhsLTF8EorPHsHEzWExhdP4pge1q9mr",
			        "datetime": "2022-10-17T15:35:43.937866Z"
		        },
		        "geometry": {
			        "type": "Polygon",
			        "coordinates": [
						[-72.34003371649325, 46.874317393516314],
			        ]
		        }
	        },
        ]
```

**Comment by matthewnanas** [comment]
Addressed in [019a8d7e5469809b3f289e79964175bcc0a9fc1e](https://github.com/easierdata/web3-geo-dashboard/commit/019a8d7e5469809b3f289e79964175bcc0a9fc1e)

**Comment by matthewnanas** [comment]
Sorry about that, the retrieval script has been updated in [my latest commit](https://github.com/easierdata/web3-geo-dashboard/commit/23425c83d40e46a484d953645a5033e637c99a13)
